dev/skeletons/skeleton_class.class.php

```diff
@@ -175,6 +178,69 @@ function fetch($id,$ref='')
         }
     }
 
+    /**
+     *  Load object in memory from the database
+     *
+     * @param string $sortorder Sort Order
+	 * @param string $sortfield Sort field
+	 * @param int $limit offset limit
+	 * @param int $offset offset limit
+	 * @param array $filter filter array
+	 * @return int <0 if KO, >0 if OK
+     */
+    function fetchAll($sortorder, $sortfield, $limit, $offset, $filter = array())
```

Comment:
Most read permission are done in card or list pages, eg fetch method (no user rights test)